index.bs

```diff
@@ -126,6 +133,7 @@ For additional terms and idioms specific to the [[PRESENTATION-API|Presentation
 API]] or [[REMOTE-PLAYBACK|Remote Playback API]], please consult the respective
 specifications.
 
+Issue(144): Receiver/Controller/Agent terminology
```

Comment:
Nit: should issues lines have a period or not? I would vote for consistency. Should be pretty easy with a regex, something like
```
find: ( *Issue[^\n]*[^.]{1})\n
replace: $1.\n
```